src/main/kotlin/com/squareup/cash/hermit/goland/HermitGoEnvUpdater.kt

```diff
@@ -20,6 +20,7 @@ class HermitGoEnvUpdater : HermitPropertyHandler {
 
             if (
                 hermitPackage.version != sdk.version
+                || hermitPackage.goURL() != sdk.homeUrl
```

Comment:
`Paths.equals()` as well to handle case insensitivity/etc...

Reply:
That is an URL (don't ask me why) so we can't use Path here, I think